### PR TITLE
Update DAY-02.md

### PR DESCRIPTION
**Pull request by IndAlok**
Hehe boi 😉

### CODE REVIEW

File: DAY-02.md [hunk 1]
```diff
@@ -7,36 +7,36 @@
 <a href="https://t.me/joinchat/xP-wW-A5mIBmMjY1"><img title="Telegram" src="https://img.shields.io/badge/Telegram-black?style=for-the-badge&logo=Telegram"></a>
 <a href="https://t.me/Mods_byAbhi_demandbot"><img title="Telegram Bot" src="https://img.shields.io/badge/Telegram-bot-black?style=for-the-badge&logo=Telegram_bot"></a></p>
 
-### continuing Day 01
+### Continuing Day 01
 
-_So I left somethings on DAY 01 because I want you to not get forget the things and that's why left in middle so when I start from left part you need to recall previous ones_ 
+_So I left some things on DAY 01 because I don't you to forget the things and that's why left in middle so when I start from left part you need to recall previous ones_
```
Comment (AbhiTheModder) [inline DAY-02.md]: 😑😑

Comment (AbhiTheModder) [reply DAY-02.md]: If you want to modify it like this then it should be "don't want"

**Review by AbhiTheModder — CHANGES_REQUESTED** [review]
¯⁠\⁠_⁠ಠ⁠_⁠ಠ⁠_⁠/⁠¯

File: DAY-02.md [hunk 2]
```diff
@@ -7,36 +7,36 @@
...
+_It denotes that a method does not have a return type._ **But this is also a point to note that a constructor method can never have a return type, but it does not have the void keyword in its declaration** So don't think that constructor will also have )V at end even when it can't have return types. For more info you should consider learning Java not smali because you're not going to write programs just remember that void used only to denote that the method has no return types. And so whereever you'll see a void method, notice that before `.end method` there's always `return-void` to return with.
   - In smali it's denoted by **V** and in java you've to write **void**
   - That's why in DAY 01 example java code declares **void** as a word but in smali it became just a single word **V**
 
   - **Now VOID are of two types one is _void type_ and another is _void keyword_
   - The first line which I talked about is _void keyword_ 
-  - The _void type does the work like an end method as we say that method is codes starting parent and .end method is it's ending he same way void type represent the void return type as a class and contain a Class<Void> public value which you've seen.
```
Comment (AbhiTheModder) [inline DAY-02.md:29]: Wrong!

File: DAY-02.md [hunk 3]
```diff
@@ -53,70 +53,70 @@ _It denotes that a method does not have a return type._ **But this is also a poi
...
 
-   - Here, you can see that void is returning means the registers used here is being returned so if we want to disable it we can add `const/4 v0, 0x0` before return method and the return method will do its function
-   - this is point to note that I take this code just as an example maybe if you modify the hello world this example then it not work or crashes i just taken it as an example to tell what it actually does.
+   - Here, you can see that void is returning, means the registers used here is being returned. So if we want to disable it, we can add `const/4 v0, 0x0` before return method, and the return method will do its function.
+   - This is a point to note that, I take this code just as an example. Maybe if you modify the hello world in this example, then it won't work or crashes. I just took it as an example to tell what it actually does.
 
-### _Now let's talk about these v0,v2,v2,v3,v4p0,p1,p2 blah blah_
+### _Now let's talk about these v0,v1,v2,v3,v4,p0,p1,p2 blah blah_
       These are called Registers names
-    so, first talk about registers
```
Comment (AbhiTheModder) [inline DAY-02.md:61]: Could you please modify this to "So, let's first talk about registers"

File: DAY-02.md [hunk 4]
```diff
@@ -53,70 +53,70 @@ _It denotes that a method does not have a return type._ **But this is also a poi
...
+   * Its alternative is _.locals_ which specifies the **non-parameters** in the code.
+   * Now go and read <a href="https://github.com/JesusFreke/smali/wiki/Registers"> This wiki</a> two-three-four as many times, until you don't understand it. If you're still not able to, then message at <a href="https://t.me/ReverseEngineering0">Telegram group</a>, I'll see where you're facing the issue. Then I'll reply you and next day I'll mention that in wiki so others don't face the same issue in future 😀
+   * As per user requests, I explained it further from DAY 5, so have a look on them too :)
 
-**Now I've told every basic thing, for further understanding you should consider reading the smali grammar table don't learn the table each time you modify the app for understanding read the table by every time reding you'll yourself don't worry so much take small steps you'll find table below😊😊**
+**Now I've told every basic thing, for further understanding you should consider reading the smali grammar table. Don't learn the table. Each time you're modifying any application, have a look on the table. Don't worry, gradually you'll learn it. You'll find table below 😊😊**
```
Comment (AbhiTheModder) [inline DAY-02.md]: Instead of "You'll find ..." Please correct it to "You can find..."

File: DAY-02.md [hunk 5]
```diff
@@ -53,70 +53,70 @@ _It denotes that a method does not have a return type._ **But this is also a poi
...
-         So, In this apk I'm going to search for remove ads 
+      3. Extract the app into apk or you can download apk of apps from various internet sources.
+      4. Try to search for the things which you saw in the app for which it was asking to pay. **Remember, search that as strings. If you've android, you can use MT Manager etc. like tools which allows you to search for specially strings but if you're on windows you can use "your_word" to search. Remember to use these quotation marks ("), before and after your word, so that any code editor, search for the strings only. I'm using MT Manager here.
+         So, in this apk, I'm going to search for **remove ads** 
 ![Screenshot 2022-02-15 135752](https://user-images.githubusercontent.com/85984486/154022556-1040d79a-071e-4a44-b2ed-673c9210a09f.png)
 The result is:
 ![image](https://user-images.githubusercontent.com/85984486/154022653-a8d6e30d-dc23-4c89-b9c6-9f8f0a63da92.png)
-_**But the result's doesn't seems interesting to me so I'll try to search for the words separately**_
+_**But the resultss don't seem interesting to me, so I'll try to search for the words separately**_
```
Comment (AbhiTheModder) [inline DAY-02.md]: extra "s" in results

**Review by AbhiTheModder — CHANGES_REQUESTED** [review]

**Review by IndAlok — COMMENTED** [review]
:)

**Review by IndAlok — COMMENTED** [review]
maybe done :)

File: DAY-02.md [hunk 6]
```diff
@@ -7,41 +7,42 @@
...
   - In smali it's denoted by **V** and in java you've to write **void**
   - That's why in DAY 01 example java code declares **void** as a word but in smali it became just a single word **V**
 
   - **Now VOID are of two types one is _void type_ and another is _void keyword_
   - The first line which I talked about is _void keyword_ 
-  - The _void type does the work like an end method as we say that method is codes starting parent and .end method is it's ending he same way void type represent the void return type as a class and contain a Class<Void> public value which you've seen.
-  - Therefore, the only value we can assign to a Void variable is null
+  - The _void type works like an end method. Just like we say that `.method` is codes(in Smali a method directive) starting parent notation, the `.end` method is it's ending,  similarly, void type represents the void return type as a class and contain a Class<Void> public value which you've seen.
```
Comment (AbhiTheModder) [inline DAY-02.md]: In Smali end method line is concatenated as `.end method`

**Review by AbhiTheModder — CHANGES_REQUESTED** [review]

**Review by AbhiTheModder — APPROVED** [review]

**Pull request merged**